### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by anirudhdmenon**
Corrected some Grammar mistakes and errors in the readme file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,8 +42,7 @@ repositories {
 
 #### 2. Add ExoPlayer module dependencies ####
 
-Next add a dependency in the `build.gradle` file of your app module. The
-following will add a dependency to the full library:
+Next, add a dependency in the `build.gradle` file of your app module. The following will add a dependency to the full library:
```
Comment (AquilesCanta) [inline README.md:45]: The line-break should stay in order to keep the column count limit.

File: README.md [hunk 2]
```diff
@@ -91,7 +90,7 @@ If not enabled already, you also need to turn on Java 8 support in all
 
 ```gradle
 compileOptions {
-  targetCompatibility JavaVersion.VERSION_1_8
+  targetcompatibility JavaVersion.VERSION_1_8
```
Comment (AquilesCanta) [inline README.md:93]: what's the rationale for this change?

**Review by AquilesCanta — COMMENTED** [review]
I think you'll need to send this PR to dev-v2 for it to be considered. Then it might make sense to cherrypick into the release branch.

**Comment by anirudhdmenon** [comment]
> @googlebot I signed it!
